tests/test_embed/test_interpreter.cpp

```diff
@@ -197,6 +197,22 @@ TEST_CASE("Custom PyConfig with argv") {
         const auto &cpp_widget = py_widget.cast<const Widget &>();
         REQUIRE(cpp_widget.argv0() == "a.out");
     }
+
+    {
+        PyConfig config;
+        PyConfig_InitPythonConfig(&config);
+
+        // Parse arguments the way python normally does, and strip the first argument (which is
+        // expected to be "python")
+        char *argv[] = {strdup("python"), strdup("a.out")};
```

Comment:
Could you please change this to

```
char *argv[] = {strdup("a.out"), strdup("arg1")};
```

I'd be easily confused otherwise seeing this in passing. "a.out" is usually only used as a name for an executable.

Reply:
Amended in 20b138d77c002973ad369c30c5bd56fde24f2232